include/zephyr/linker/common-rom/common-rom-net.ld

```diff
@@ -27,5 +27,5 @@
 #endif
```

Comment:
Just noticed, that perhaps add "Fixes ..." to the commit message.

Reply:
Ok, updated the commit message.